src/components/dialogs/parameters/security-analysis/security-analysis-parameters.tsx

```diff
@@ -0,0 +1,206 @@
+    useSnackMessage,
+} from '@gridsuite/commons-ui';
+import { fetchSecurityAnalysisParameters } from '../../../../services/security-analysis.js';
+import SecurityAnalysisParametersSelector from './security-analysis-parameters-selector';
+import CreateParameterDialog from '../common/parameters-creation-dialog';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+    getLimitReductionsFormSchema,
+    ILimitReductionsByVoltageLevel,
+    IST_FORM,
+    LIMIT_DURATION_FORM,
+    LIMIT_REDUCTIONS_FORM,
+} from './columns-definitions';
+
+export const SecurityAnalysisParameters: FunctionComponent<{
+    parametersBackend: any[];
+    setHaveDirtyFields: Dispatch<SetStateAction<boolean>>;
+}> = ({ parametersBackend, setHaveDirtyFields }) => {
+    const [providers, provider, updateProvider, resetProvider, params, updateParameters, resetParameters] =
+        parametersBackend;
+
+    const handleUpdateProvider = (evt: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) =>
+        updateProvider(evt.target.value);
+
+    const updateProviderCallback = useCallback(handleUpdateProvider, [updateProvider]);
+    const intl = useIntl();
+    const [openCreateParameterDialog, setOpenCreateParameterDialog] = useState(false);
+    const [openSelectParameterDialog, setOpenSelectParameterDialog] = useState(false);
+    const { snackError } = useSnackMessage();
+
+    // TODO: remove this when DynaFlow is supported
+    // DynaFlow is not supported at the moment for security analysis
+    const securityAnalysisProvider = Object.fromEntries(
+        Object.entries(providers).filter(([key]) => !key.includes('DynaFlow'))
+    );
+
+    const resetSAParametersAndProvider = useCallback(() => {
+        resetParameters();
+        resetProvider();
+    }, [resetParameters, resetProvider]);
+
+    const resetSAParameters = useCallback(() => {
+        resetParameters();
+    }, [resetParameters]);
+
+    const handleLoadParameter = useCallback(
+        (newParams: Record<string, any>) => {
+            if (newParams && newParams.length > 0) {
+                setOpenSelectParameterDialog(false);
+                fetchSecurityAnalysisParameters(newParams[0].id)
+                    .then((parameters) => {
+                        console.info('loading the following security analysis parameters : ' + parameters.uuid);
+                        updateParameters({ ...parameters });
+                    })
+                    .catch((error) => {
+                        console.error(error);
+                        snackError({
+                            messageTxt: error.message,
+                            headerId: 'paramsRetrievingError',
+                        });
+                    });
+            }
+            setOpenSelectParameterDialog(false);
+        },
+        [snackError, updateParameters]
+    );
+
+    const formSchema = getLimitReductionsFormSchema(params.limitReductions);
```

Comment:
I'm not sure how useForm will behave here, but I think I would use "useMemo" to prevent recreating new "formSchema" on each render